include/swift/Basic/Range.h

```diff
@@ -134,7 +134,7 @@ class IntRange {
       return *this;
     }
     iterator operator+(difference_type i) const {
-      return iterator(Traits::adddOfset(Value, i));
+      return iterator(Traits::adddOffset(Value, i));
```

Comment:
??

Reply:
This should be `Traits::addOffset`. This template member is not being instantiated so we never noticed the error...